Update homepage to show benefits

frankieroberto opened the pull request:
This updates the homepage to more clearly show the benefits of using the NHS prototype kit.

Resolves #50. 

Inspiration taken from the [NHS Notify](https://notify.nhs.uk) homepage and [a previous mockup](https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/pull/89#issuecomment-2461609758) from @vickytnz.

## Screenshot

![prototype-kit-homepage](https://github.com/user-attachments/assets/4a59f74e-fbc9-4788-9d6c-1c6ba1c09816)

paulrobertlloyd commented on app/views/index.html:
```diff
@@ -26,44 +32,20 @@ <h1 class="nhsuk-heading-l">
-
+    <div class="nhsuk-grid-column-one-third">
+      <h2 class="nhsuk-heading-m">Realistic</h2>
+      <p>Create a prototype that behaves like the real thing, across devices, using components and styles from the NHS Design System.
```
Elsewhere we say ‘Use prototypes’ and ‘Test designs’, so the singular here stands out a bit.

```suggestion
      <p>Create prototypes that behave like the real thing, across devices, using components and styles from the NHS Design System.
```

sarawilcox commented:
I like it.

We wouldn't usually include adjectives like "valuable" but I see that GOV does and here we're trying to persuade people, not just give the facts, so I think we can leave it in.

I wonder if we should mention that the prototypes are interactive. Under the Realistic heading, would it work to say: 

"Create interactive prototypes that behave like the real thing, across devices, using styles and components from the design system"

frankieroberto commented:
@sarawilcox nice idea, I like 'interactive' as it's a point of differentiation between static designs done in an illustration tool.

Here’s how that looks (I dropped "styles and" to make it fit): 

<img width="1050" alt="Screenshot 2025-02-19 at 14 14 16" src="https://github.com/user-attachments/assets/91bd4bfb-f319-4abb-915e-c18dae7abcef" />

I’m not quite sure about "across devices" – I added that to try and imply that it’ll work on mobile/tablets/desktop/etc, but in fewer words. Is there a better way to say this?
